src/overlays/actors/ovl_En_S_Goro/z_en_s_goro.c

```diff
@@ -74,85 +144,1336 @@ static DamageTable D_80BBF858 = {
+    { &object_taisou_Anim_005790, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
+    { &gGoronCoverEarsAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
+    { &object_taisou_Anim_002C48, 1.0f, 0, -1, ANIMMODE_LOOP, -4 },
+    { &gGoronStandingHandTappingAnim, 1.0f, 0, -1, ANIMMODE_LOOP, -4 },
+    { &gGoronSleepyAnim, 1.0f, 0, -1, ANIMMODE_ONCE, -4 },
+    { &gGoronStandingIdleAnim, 1.0f, 0, -1, ANIMMODE_LOOP, -8 },
+};
+
+static TexturePtr sEyeTextures[] = {
+    gGoronEyeOpenTex,
+    gGoronEyeHalfTex,
+    gGoronEyeClosedTex,
+    gGoronEyeClosed2Tex,
+};
+
+u16 EnSGoro_ShrineGoron_NextTextID(EnSGoro* this, PlayState* play);
+u16 EnSGoro_BombshopGoron_NextTextID(EnSGoro* this, PlayState* play);
+
+s32 EnSGoro_SetPartnerActor_GoronElder(EnSGoro* this, PlayState* play);
+s32 EnSGoro_SetPartnerActor_GoronChild(EnSGoro* this, PlayState* play);
+s32 EnSGoro_CheckLullaby(EnSGoro* this, PlayState* play);
+s32 EnSGoro_CheckGKBehavior(EnSGoro* this, PlayState* play);
+
+void EnSGoro_SetupAction(EnSGoro* this, PlayState* play);
+void EnSGoro_WinterShrineGoron_Idle(EnSGoro* this, PlayState* play);
+void EnSGoro_WinterShrineGoron_Talk(EnSGoro* this, PlayState* play);
+void EnSGoro_SpringShrineGoron_Idle(EnSGoro* this, PlayState* play);
+void EnSGoro_SpringShrineGoron_Talk(EnSGoro* this, PlayState* play);
+void EnSGoro_ShopGoron_Idle(EnSGoro* this, PlayState* play);
+void EnSGoro_ShopGoron_FinishUnroll(EnSGoro* this, PlayState* play);
+void EnSGoro_ShopGoron_Talk(EnSGoro* this, PlayState* play);
+void EnSGoro_ShopGoron_TakePayment(EnSGoro* this, PlayState* play);
+void EnSGoro_ShopGoron_FinishTransaction(EnSGoro* this, PlayState* play);
+void EnSGoro_SleepAction(EnSGoro* this, PlayState* play);
+void EnSGoro_SleepTalkAction(EnSGoro* this, PlayState* play);
+
+s32 EnSGoro_UpdateCheerAnimation(EnSGoro* this, PlayState* play);
+s32 EnSGoro_UpdateRotationToPlayer(EnSGoro* this, PlayState* play);
+s32 EnSGoro_UpdateAttentionTarget(EnSGoro* this, PlayState* play);
+void EnSGoro_UpdateToHandtapAnimation(EnSGoro* this);
+void EnSGoro_UpdateSleepyAnimation(EnSGoro* this);
+void EnSGoro_UpdateToIdleAnimation(EnSGoro* this);
+void EnSGoro_UpdateEyes(EnSGoro* this);
+void EnSGoro_UpdateActorFocus(EnSGoro* this);
+void EnSGoro_UpdateSleeping(EnSGoro* this, PlayState* play);
+void EnSGoro_UpdateCollider(EnSGoro* this, PlayState* play);
+
+s32 EnSGoro_UpdateLimb(s16 newRotZ, s16 newRotY, Vec3f* pos, Vec3s* rot, s32 stepRot, s32 overrideRot);
+s32 EnSGoro_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx);
+void EnSGoro_TransformLimbDraw(PlayState* play, s32 limbIndex, Actor* thisx);
+void EnSGoro_DrawUnrolled(EnSGoro* this, PlayState* play);
+void EnSGoro_DrawRolledUp(EnSGoro* this, PlayState* play);
+
+/**
+ * Dialogue tree for EnSGoro when use in Goron Shrine context. Returns ID of next message to display.
+ */
+u16 EnSGoro_ShrineGoron_NextTextID(EnSGoro* this, PlayState* play) {
+
+    Player* player = GET_PLAYER(play);
```

Comment:
```suggestion
u16 EnSGoro_ShrineGoron_NextTextId(EnSGoro* this, PlayState* play) {
    Player* player = GET_PLAYER(play);
```
No spaces between the function declaration and variable initailization